tests/benchmarks/multicore/idle_pwm_loopback/testcase.yaml

```diff
@@ -0,0 +1,88 @@
+common:
+  sysbuild: true
+  tags: ci_build ci_tests_benchmarks_multicore
+  platform_allow:
+    - nrf54h20dk/nrf54h20/cpuapp
+  integration_platforms:
+    - nrf54h20dk/nrf54h20/cpuapp
+
+tests:
+  benchmarks.multicore.idle_pwm_loopback.nrf54h20dk_cpuapp_cpurad.no_sleep:
+    extra_args:
+      SB_CONF_FILE=sysbuild/nrf54h20dk_nrf54h20_cpurad.conf
+    harness: console
+    harness_config:
+      fixture: spi_loopback
+      type: multi_line
+      ordered: true
+      regex:
+        - "Multicore idle_pwm_loopback test on"
+        - "Iteration 0"
+        - "Iteration 1"
+
+  benchmarks.multicore.idle_pwm_loopback.nrf54h20dk_cpuapp_cpurad.idle:
+    tags: ppk_power_measure
+    extra_args:
+      SB_CONF_FILE=sysbuild/nrf54h20dk_nrf54h20_cpurad.conf
+      idle_pwm_loopback_CONF_FILE=prj_s2ram.conf
+      remote_CONF_FILE=prj_s2ram.conf
+      idle_pwm_loopback_CONFIG_TEST_SLEEP_DURATION_MS=500
+      remote_CONFIG_TEST_SLEEP_DURATION_MS=500
+    harness: pytest
+    harness_config:
+      fixture: spi_loopback
+      pytest_root:
+        - "${CUSTOM_ROOT_TEST_DIR}/test_measure_power_consumption.py::test_measure_and_data_dump_pwm_and_idle"
+
+  benchmarks.multicore.idle_pwm_loopback.nrf54h20dk_cpuapp_cpurad.s2ram:
+    tags: ppk_power_measure
+    extra_args:
+      SB_CONF_FILE=sysbuild/nrf54h20dk_nrf54h20_cpurad.conf
```

Comment:
can be removed if following the remote board suggestion.
https://github.com/nrfconnect/sdk-nrf/pull/19018#discussion_r1853707524

Reply:
All occurrences of `SB_CONF_FILE=sysbuild/nrf54h20dk_nrf54h20_cpurad.conf` were removed from `extra_args`.